README.md

```diff
@@ -21,7 +21,8 @@
         <b>English</b> |
         <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_de.md">Deutsch</a> |
         <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md">हिंदी</a> | 
-        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a>
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a> |
+         <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_cn.md">中文（简体）</a> 
```

Comment:
Nice to add a `_cn` README! Could you also add the link to other READMEs? ([README_de.md](https://github.com/huggingface/huggingface_hub/blob/main/README_de.md), [README_ko.md](https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md) and [README_hi.md](https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md)). Thanks in advance!

Reply:
No problem, I will add it when I have some free time🦾. Thanks!@ Wauplin

Reply:
Chinese readme links have been added to readme files in other languages @Wauplin